tests/security/selinux/selinux_setup.pm

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup environment for selinux test
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#40358
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use version_utils "is_opensuse";
+
+sub run {
+    select_console "root-console";
+
+    if (is_opensuse) {
+        zypper_call("ar http://download.opensuse.org/tumbleweed/repo/oss/ tumbleweed-Oss");
```

Comment:
looks strange :
1. you check `is_opensuse` which means that it will pass for Leap 15 for example too, but you adding TW repo 
2. this is standard TW repo which you getting by default with TW so point to re-add it ?

Reply:
1. It only supports opensuse tumbleweed now.
2. deleted, thanks for the pointing.